localisation/00_wc_random_artifact_names.csv

```diff
@@ -0,0 +1,463 @@
+#CODE;ENGLISH;;;;;;;;;;;;;x
+#Fallback text;;;;;;;;;;;;;;x
+artifact_jewelry_common_prefix_0;Common;;;;;;;;;;;;;x
+artifact_jewelry_uncommon_prefix_0;Uncommon;;;;;;;;;;;;;x
+artifact_jewelry_rare_prefix_0;Rare;;;;;;;;;;;;;x
+artifact_jewelry_epic_prefix_0;Mythical;;;;;;;;;;;;;x
+artifact_jewelry_legendary_prefix_0;Legendary;;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
+artifact_jewelry_uncommon_suffix_0;[Nothing];;;;;;;;;;;;;x
+artifact_jewelry_rare_suffix_0;[Nothing];;;;;;;;;;;;;x
+artifact_jewelry_epic_suffix_0;[Nothing];;;;;;;;;;;;;x
+artifact_jewelry_legendary_suffix_0;[Nothing];;;;;;;;;;;;;x
+;;;;;;;;;;;;;;x
```

Comment:
See comments on spreadsheet for renaming/corrections

Reply:
Changed!
See spreadsheet for more info